docs/notebooks/autodiff_remat.md

```diff
@@ -0,0 +1,548 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
```

Comment:
You need a top-level markdown heading at the top of this notebook to be the TOC title... otherwise sphinx does weird things with the table of contents.

Reply:
Bleh. I wanted to have the "utilities" section collapsed at the top _before_ the title. But I guess that's not possible...